workers/tests/integrationtests/api/test_metadata_topics.py

```diff
@@ -36,7 +36,7 @@ def test_metadata_topic(client, response_template, building_count):
 
 
 @pytest.mark.skip(reason="Not yet implemented")
-def test_metadata_topic_project_core(client, response_template, building_count):
+def test_project_core(client, response_template, building_count):
```

Comment:
see above. What was wrong with the more detailed name?

Reply:
The rationale was avoiding the information present in the name of the test file `test_metadata_topics`